packages/@okta/i18n/src/properties/login_zh_CN.properties

```diff
@@ -1358,9 +1358,9 @@ idx.return.link.otponly.app = {0}
 # {0} is the name of the browser and {1} is the name of the devices OS that triggered the login
 idx.return.link.otponly.browser.on.os = {1} \u5728 {0}
 # {0} is the city, {1} is the state, and {2} is the country from where the login request originated
-geolocation.formatting.all = {0}, {1}, {2}
+geolocation.formatting.all = {2}{1}{0}
 # {0} is the city and {1} is the country from where the login request originated
-geolocation.formatting.partial = {0}, {1}
+geolocation.formatting.partial = {1}{0}
```

Comment:
Same.

Reply:
Other languages have a separator between city and county. I think this should be reverted.
If it is the correct format, then I think zh_TW should be changed as well?

Reply:
We discussed this here : https://okta.slack.com/archives/C033FGSMNDN/p1674843331901639?thread_ts=1674838977.900509&cid=C033FGSMNDN 

Will not revert for now since it doesn't seem to affect readability in Chinese?